src/js/components/molecules/cards/card-article/card-article.models.ts

```diff
@@ -0,0 +1,9 @@
+/** @format */
+
+import { CardBaseProps } from 'components/molecules/cards/card-base/card-base.models';
+import { CardArticle } from '@ultimaker/ultimaker.com-model-definitions/dist/molecules/cards/CardArticle';
+
+export interface CardArticleProps extends CardBaseProps {
```

Comment:
``` export interface CardArticleProps extends CardBaseProps, CardArticle {}```

Reply:
tried it out but a few errors appeared. we can look at this together a bit later.